bugsnag-android-core/src/main/java/com/bugsnag/android/Session.java

```diff
@@ -12,12 +12,14 @@
 /**
  * Represents a contiguous session in an application.
  */
+@SuppressWarnings("ConstantConditions")
 public final class Session implements JsonStream.Streamable, UserAware {
```

Comment:
Shouldn't this have a `SessionInternal` backing class as per the others?

Reply:
The `Session` class has not been implemented in Kotlin so there's no need to do this.